### DISCUSSION TITLE
feature proposal: diagrams.net integration

### DISCUSSION THREAD

**Discussion by bboh** [outline/outline#1408] [General]
https://www.diagrams.net/
It's a really good OSS alternative to Lucidchart

**Comment by matbrgz** [comment]
Related to https://github.com/outline/outline/issues/820

**Comment by tommoor** [comment]
Looks good, do you have any examples of these being embedded?

**Comment by wvds** [comment]
@tommoor I've created PR #2464 with the implementation for Diagrams.net embeds, also with an example URL to test.
